### ISSUE TITLE
Results Type CV vs. Results Structure

### ISSUE THREAD

**Issue by SRGDamia1** [UCHIC/ODM2#84]
We have more allowable result types than have places to be stored in the results schema. Where do the other result types belong?

ie:

Result Type Term                        Appropriate Result Value Table
Measurement                              MeasurementResultValues
Temporal Observation                 ??????????
Count Observation                      ??CategoricalResultValues??
Truth Observation                       ??CategoricalResultValues??
Category Observation                 CategoricalResultValues

**Comment by emiliom** [comment]
Good catch. We decided we were not going to support explicitly, via separate tables, these edge OGCM O&M result types. I can't remember if we decided we weren't going to support these result types, period, or that they could be accommodated within the existing tables. If it's the latter, we need a match-up between result types and the corresponding table for these cases; if it's the former, the terms should be removed from the vocabulary. At any rate, these are likely to be rare in the CZO's.
